app/views/rubygems/show.html.erb

```diff
@@ -1,7 +1,12 @@
 <% @title = @rubygem.name %>
 <% @subtitle = @latest_version.try(:slug) %>
 
-<%= content_for :head, auto_discovery_link_tag(:atom, rubygem_versions_path(@rubygem, format: "atom"), {title: "#{@rubygem.name} Version Feed"}) %>
+<% content_for :head do %>
+  <%= auto_discovery_link_tag(:atom, rubygem_versions_path(@rubygem, format: "atom"), {title: "#{@rubygem.name} Version Feed"}) %>
+  <% unless @latest_version.indexed %>
```

Comment:
@arthurnn Thanks for the feedback. I've put in an additional check similar to what is used further down the page here https://github.com/bradly/rubygems.org/blob/master/app/views/rubygems/show.html.erb#L6